packages/root/src/core/hooks/useTranslations.ts

```diff
@@ -26,6 +26,11 @@ export function useTranslations() {
   const stringParams = useStringParams();
   const middleware = useTranslationMiddleware();
   const t = (str: string, params?: Record<string, string | number>) => {
+    // Suppress verbatim `'undefined'` and `'null'` output, which can occur when
+    // using `useTranslations` with undefined values.
+    if (typeof str === 'undefined' || str === null) {
+      return undefined;
```

Comment:
i think if null is passed maybe null should be returned, so change `return undefined` to `return str`.

Reply:
done; added comments